packages/backend/test/utils.ts

```diff
@@ -123,17 +123,18 @@ export const getSpent = async (t, iden) => {
         )
     }
     const paramStr = epks.join('_')
-    const r = await fetch(
-        `${t.context.url}/api/records/${paramStr}?spentonly=true`
-    )
+    const r = await fetch(`${t.context.url}/api/records/${paramStr}`)
     const data = await r.json()
     if (!r.ok) {
         throw new Error(`/records error ${JSON.stringify(data)}`)
     }
```

Comment:
We should also probably do the error handling for the response right after we are awaiting the fetch.
So, it would be:
```js
const r = await fetch (..)
if (!r.ok) { .. throw the error}
const data = await r.json()
```
I think in the current form, we would not get the error handling we expect.

Reply:
The current form for error handling is also being used in other functions within this file.